__tests__/app.test.js

```diff
@@ -116,6 +132,41 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Returns all comments based on article_id, descending order", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+          expect(comments.length).toBeGreaterThan(0);
+
+          expect(comments[0].article_id).toBe(1);
+
+          expect(comments).toBeSorted({ descending: true });
```

Comment:
what should it be sorted by as default? - look into the toBeSortedBy() matcher from jest sorted

Reply:
As this tests a different behaviour, we can also put it into its own test!